### DISCUSSION TITLE
Working on maps saved in Google Drive (cloud backup)

### DISCUSSION THREAD

**Discussion by ArNicoS** [freeplane/freeplane#1000] [Help]
Hello, 

I would like to work on Freeplane maps (.mm) files that are located in Google Drive selected for automatic backup folders. This is to allow me to access them on different computer and have always safely stored in a place not subject to hardware malfuction (my PC temporary failure, etc.)

I would like to share with the community the experience I had when saving freplane maps in such location and eventually ask your help for solving some related issues.

First thing a information message I get every time I save a map in Google Drive. I attached a screenshot of it. Basically, just after saving the map I get a information message stating that "the map file was changed by another program".  Even if I tick the box for "don't show it again the message appears everytime I save the same map or any map stored in Google Drive.
![image](https://user-images.githubusercontent.com/119033438/218038668-dd4db764-1e0f-49bd-85b2-9d128d03c09e.png)

Second thing I notice few days later I set links to other files (other Freeplane maps)   in some nodes of a map stored in the cloud, in this case Google Drive, such link get broken. I thought this might be related to the fact that Google Drive is "changing" the maps I am trying to access via the link added in other map. 

What do your think? Are you aware of any issue like this when working on maps stored in Google Drive selected for upload folders or similar cloud services susch as OneDrive, Drobbox etc.? 

Do you have any recommendation for setting up an automatically cloud backed up folder where to keep the mind maps you are creating with Freeplane?

Thank you for your insights.

**Comment by bepolymathe** [comment]
Hi @ArNicoS 

I always use a configuration of this type with Nextcloud without any problem. 

What are the types of machines from which you access your maps ? Are all the machines under windows ? Do the links point to files that are in the same synchronized folder? Are the links "absolute" or "relative" in your configuration ? (tools --» preferences --» environment --» hypertext links)

**Comment by ldmpub** [comment]
@ArNicoS to answer your first remark above
> First thing an information message I get every time I save a map in Google Drive

I confirm than I'm running Freeplane with *.mm files stored in Google Drive without any inconvenience. My context: Windows 10 + Freeplane 1.10.6 + Google Drive Sync Tool.

I remember that, in the past, I got the same message as you but it has vanished. **Maybe upgrading Freeplane will help you solve this problem?** Otherwise I'm ready to have a look in Preferences to compare my active options with yours.

**Comment by ArNicoS** [comment]
Hello @bepolymathe  and @ldmpub  thank  you for your replies.

Over the past week I kept testing the saving of .mm files in google drive selected for upload folders. I have not experienced anymore issues with cross-linking .mm files by embedding links in the map. However, the window with the message reporting the map file being modified by another program is still coming even after ticking the box for "don't show anymore".

@bepolymathe 
I am accessing the maps file always with the same laptop. OS is windows 11. 
The links point to maps that are stored in the same Google drive folder.  I have not tried with maps stored in different folders recently. Do you whether it is possible to store the maps in different folders or it is going to be an issue?
The hyperlink type is "relative".
![image](https://user-images.githubusercontent.com/119033438/221800980-06cf49cd-eefe-406e-9a8e-2c3860c46321.png)

@ldmpub 
I am using Freeplane v 1.10.6 + Google drive 71.0.3.0
![image](https://user-images.githubusercontent.com/119033438/221801374-a5068f48-8c2b-4e6e-b444-a868d4ecdbf8.png)
![image](https://user-images.githubusercontent.com/119033438/221801828-9a601b4f-0196-4239-a56a-c09d7e82239f.png)

Can you suggest an efficient way to share Freeplane preferences?

Thank you both!

**Comment by xc3seziu** [comment]
Windows 11 with all updates installed.
Freeplane 1.11.11.
Google Drive 88.0.0.0 using Stream to sync.

I just started using Google Drive for Freeplane maps and am running into this problem. It started right after I created and saved a new map that didn't yet have any content.

Is there any additional information I can provide regarding this issue? It doesn't affect functionality or cause any other problems, it's just an annoyance for this dialog to appear every time I save the file. lol
